bruker2nifti/cli/bruker2nii.py

```diff
@@ -159,5 +201,27 @@ def main():
         print("INFO: Output path/filename contains whitespace")
 
 
+def list_scans(pfo_study):
+    study = BrukerMetadata(pfo_study)
+    study.parse_subject()
+    study.parse_scans()
+
+    # Print study details to the console
+    print()
+    print("Subject: {}".format(study.subject_data["SUBJECT_name_string"]))
+    print("Study Date: {:%Y-%m-%d %H:%M}".format(datetime.strptime(
+      study.subject_data["SUBJECT_date"], "%Y-%m-%dT%H:%M:%S,%f%z")))
+    print("-------------------------------------------------------- ")
+    print()
+    for scan, value in study.scan_data.items():
+        print("Scan {}".format(scan))
+        print(
+          "Protocol: {}".format(value["acqp"]["ACQ_protocol_name"]).ljust(30) +
+          "Method: {}".format(value["acqp"]["ACQ_method"]).ljust(30) +
+          "Scan Time: {}".format(timedelta(
+          seconds=int(round(value["method"]["ScanTime"] / 1000)))).ljust(30))
+        print()
+
+
```

Comment:
Cool! But CLI should have only the CLI method. What about moving this under metadata?

Reply:
So my rationale for putting it here is that the output here is probably specific to the chosen UI. In a graphical frontend for example, it might be better to display the scans as a clickable list within a window.

I'd like to keep any user interface/console output outside of the metadata file and leave that to just do the actual parsing. I could potentially move this to a "console" file for example and that way it can be shared by the GUI frontends if they want to but could also be avoided if they want to use a different method of listing scans.

Reply:
Agree! Also the print used in the whole CLI is a bit naive. It make sense for now, but in the future maybe we should change it to something more interesting, as a `logging`. Good for now!